Gemfile

```diff
@@ -2,18 +2,19 @@ source 'https://rubygems.org'
 
 gem 'aws-sdk'
 gem 'carrierwave', '~> 0.10.0'
-gem 'carrierwave-mongoid', '~> 0.8.1', require: 'carrierwave/mongoid'
-gem 'gds-sso', '~> 11.2'
+gem 'carrierwave-mongoid', '~> 0.10.0', require: 'carrierwave/mongoid'
+gem 'gds-sso', '~> 13.2'
```

Comment:
Could this gem upgrade be made as a separate commit before the upgrade to Rails?